code/src/Components/ThoughtForm.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const ThoughtForm = ({ onFormSubmit, newThought, setNewThought }) => {
+  return (
+    <form className="form card" onSubmit={onFormSubmit}>
+      <label htmlFor="newThought">What's making you happy right now?</label>
+      <textarea
+        className={newThought.length > 140 ? "red-text" : ""}
+        id="newThought"
+        type="text"
+        rows="5"
+        columns="150"
+        value={newThought}
+        onChange={(e) => setNewThought(e.target.value)}
+        placeholder="Write your happy thought here..."
```

Comment:
Nice detail with the placeholder! I missed that so I´m gonna steal your idea here:)